staff/views/users.py

```diff
@@ -0,0 +1,57 @@
+import structlog
+from django.core.exceptions import ValidationError
+from django.db.models import Q
+from django.db.models.functions import Lower
+from django.http import Http404
+from django.utils.decorators import method_decorator
+from django.views.generic import ListView, UpdateView
+
+from interactive.models import User
+
+from ..decorators import staff_required
+
+
+logger = structlog.get_logger(__name__)
+
+
+@method_decorator(staff_required, name="dispatch")
+class UserDetail(UpdateView):
+    fields = [
+        "email",
+        "name",
+        "is_staff",
+        "is_active",
```

Comment:
Could we also have job title and organisation please?

Also, `is_active` isn't currently on the template.